ui/tests/helpers/codemirror.js

```diff
@@ -18,9 +16,16 @@ export function getCodeMirrorInstance(container) {
   };
 }
 
-export default function registerCodeMirrorHelpers() {
-  registerHelper('getCodeMirrorInstance', function(app, selector) {
-    const helper = getCodeMirrorInstance(app.__container__);
-    return helper(selector);
+export default function setupCodeMirror(hooks) {
+  hooks.beforeEach(function() {
+    this.getCodeMirrorInstance = getCodeMirrorInstance(this.owner);
+
+    // Expose to window for access from page objects
+    window.getCodeMirrorInstance = this.getCodeMirrorInstance;
```

Comment:
👌 nice

Reply:
Lol, it's not my proudest code, but Mirage also does this in a less obvious place. That's all the excuse I needed 😄